content/applications/finance/accounting/getting_started/process_overview/supplier_bill.rst

```diff
@@ -1,144 +1,106 @@
-
-.. image:: supplier_bill/vendor_bill05.png
-   :align: center
+.. seealso::
+   - Tutorial `Registering a vendor bill <https://www.odoo.com/slides/slide/registering-a-vendor-bill-1683?fullscreen=1>`_
+   - :ref:`Manage-vendor-bills`
 
-To register a new vendor bill, start by selecting a vendor and inputting
-their invoice as the **Vendor Reference**, then add and confirm the product
-lines, making sure to have the right product quantities, taxes and
-prices.
+Bills creation
+==============
 
-.. image:: supplier_bill/vendor_bill01.png
-   :align: center
+From the Accounting app, vendor bills can either be created manually or automatically.
 
-Save the invoice to update the pre tax and tax amounts at the bottom of
-the screen. You will most likely need to configure the prices of your
-products without taxes as Odoo will compute the tax for you.
-
-.. note::
-    On the bottom left corner, Odoo shows a summary table of all taxes on the vendor bill.
-    In several countries, different methods are accepted to round the totals (round per line,
-    or round globally). The default rounding method in Odoo is to round the final prices
-    per line (as you may have different taxes per product. E.g. Alcohol and cigarettes).
-    However if your vendor has a different tax amount on their bill, you can change the
-    amount in the bottom left table to adjust and match.
+Manually
+--------
 
-Validate The Vendor Bill
-------------------------
+It is possible to manually create a vendor bill by going to :menuselection:`Vendors --> Bills` and
+then clicking :guilabel:`Create`.
 
-Once the vendor bill is validated, a journal entry will be generated
-based on the configuration on the invoice. This journal entry may differ
-depending on the the accounting package you choose to use.
+Automatically
+-------------
 
-For most European countries, the journal entry will use the following
-accounts:
+Vendor bills can be automatically created by **sending an email** to an :ref:`email alias
+<invoice-digitization/email-alias>` associated with the purchase journal, or by **uploading a pdf**
+in :menuselection:`Accounting app --> Vendors --> Bills` and then clicking :guilabel:`Upload`.
 
--  **Accounts Payable:** defined on the vendor form
+Bills completion
+================
 
--  **Taxes:** defined on the products and per line
+Whether you create the bill manually or if it is generated automatically, make sure the following
+fields are completed as appropriate:
 
--  **Expenses:** defined on the line item product used
+First, select the :guilabel:`Vendor`. Odoo automatically fills some fields based on previous PO or
+bills from the vendor.
 
-For Anglo-Saxon (US) accounting, the journal entry will use the
-following accounts:
+Record the order reference number in the :guilabel:`Bill Reference` field. This will enables to
+later :ref:`match <payments-matching>` the PO with the vendor bill (as the vendor bill should
+include the Vendor Reference).
```

Comment:
```suggestion
include the Vendor Reference).
```
Where would that be ? I couldn't find it on the bill, and the way it is spelled, it feels like it would be a guilabel (the capitalization)

Reply:
![image](https://user-images.githubusercontent.com/85342080/210222310-0a92823f-fd16-497e-b509-f72a6ae9d56a.png)
The order reference is provided by the vendor. If it is field in the Purchase Order, this field is automatically populated.